tests/lax_test.py

```diff
@@ -1161,6 +1161,41 @@ def testDotGeneralAgainstNumpy(self, lhs_shape, rhs_shape, dtype,
     numpy_op = lambda x, y: lax_reference.dot_general(x, y, dimension_numbers)
     self._CheckAgainstNumpy(numpy_op, op, args_maker)
 
+  @jtu.sample_product(
+      [
+          dict(
+              m=m, k=k, n=n, num_groups=num_groups,
+          )
+          for m, k, n, num_groups in [
+              (5, 4, 3, 1),
+              (10, 9, 8, 2),
+          ]
+      ],
```

Comment:
```suggestion
      [
          {'m': 5, 'k': 4, 'n': 3, 'num_groups': 1},
          {'m': 10, 'k': 9, 'n': 8, 'num_groups': 2},
      ],
```
Clearer perhaps?

Reply:
Shorter, and more readable. Done here and in the export_test.py